_src/_includes/sidenote.html

```diff
@@ -0,0 +1 @@
+<label for="{{ include.reference }}" class="margin-toggle sidenote-number"></label><input type="checkbox" id="{{ include.reference }}" class="margin-toggle"/><span class="sidenote">{{ include.content }}</span>
```

Comment:
Finally, would it be more semantic if the sidenotes were `<aside>...</aside>` instead of `<span>...</span>`?